content/getting-started/synchronize-pim-products-6x/welcome.md

```diff
@@ -0,0 +1,27 @@
+# 👋 Welcome in this tutorial!
+
+This tutorial will teach you:
+> How to synchronize PIM product data with your App
+
+## Goal
```

Comment:
Perhaps we need to explain that this guide is dedicated to Apps that need to **export** data from the PIM (e-Commerce, Syndication, CMS, print ...). Because you have also Apps that need **import** data into the PIM...
And this is not the scope of this guide. Am I wrong?